### DISCUSSION TITLE
How to use P/Invoke in PascalABC.NET?

### DISCUSSION THREAD

**Discussion by ghost** [pascalabcnet/pascalabcnet#2754] [General]
Is it even possible?

**Comment by SunSerega** [comment]
Use `external`.

**Reply by ghost** [reply]
So the syntax is like of Free Pascal? What about Marshalling? Can the same syntax for C# be used?

**Reply by SunSerega** [reply]
Yes, marshaling attributes work the same as in C#.

**Reply by ghost** [reply]
Thank you.
